build-system/conformance-config.textproto

```diff
@@ -168,6 +168,22 @@ requirement: {
 }
 
 requirement: {
+  type: BANNED_PROPERTY_READ
```

Comment:
`action-impl` adds `detail` property to some event objects. I don't believe `detail` is standard prop of event object.